src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs

```diff
@@ -2,10 +2,18 @@
 
 namespace OSPSuite.Core.Domain.Builder
 {
-   public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockFromPKSim<IndividualParameter>
+   public class OriginDataCache : Cache<string, string>
    {
-      public Cache<string, string> OriginData { set; get; } = new Cache<string, string>();
+      public OriginDataCache() : base(onMissingKey: x => string.Empty)
+      {
+      }
+
       public ValueOrigin ValueOrigin { set; get; }
-      public CalculationMethodCache CalculationMethodCache { set; get; }
+      // public CalculationMethodCache CalculationMethodCache { set; get; }
```

Comment:
The serializer in PK-Sim uses a CalcuationMethodRepository which is not shared with core. So, is this object shared with MoBi? Or is there just another serializer needed?

Reply:
I don't understand why we need this calculation method repository here.
This info is only used as display purpose only. Just add it to your ORiginDataCache